src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -27,7 +28,8 @@
     private final Projact projact;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
-    private Predicate<TagName> filteredTagPredicate = x -> true;
+    private final FilteredList<Tag> filteredTags;
+    private Predicate<TagName> filteredTagPredicate = x -> true; // TODO: Remove in v1.3
```

Comment:
so just to clarify, this would be done by kk right?

Reply:
sorry isit??? i dont think so..? do correct me if im wrong..!!